symengine/dict.h

```diff
@@ -42,6 +43,7 @@ typedef std::map<RCP<const Integer>, unsigned,
         RCPIntegerKeyLess> map_integer_uint;
 typedef std::map<unsigned, integer_class>
        map_uint_mpz;
+typedef std::map<unsigned, Expression> map_uint_Expr;
```

Comment:
See some of the examples here, https://github.com/symengine/symengine/blob/master/symengine/tests/basic/test_series_expansion_URatP.cpp#L101